### DISCUSSION TITLE
Simulations targeting RESTOM and SWCF

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#309] [Tuning parameters]
## Set of FLTHIST simulations to decrease SWCF and increase RESTOM

Last coupled run BLTHIST: 
b.e23_alpha14a.BLT1850.ne30_t232.032 ([#283](https://github.com/NCAR/amwg_dev/issues/283))
RESTOM = -1.28 W/m2

Here we performed a set of FLTHIST simulations to decrease SWCF and increase RESTOM

## Control
- **Link**: #296 
- RESTOM = 1.13 W/m^2
- SWCF = -48.6 W/m^2

## clubb_gamma_coef 
- **clubb_gamma_coef 8** =  0.27 -> 0.30
- RESTOM = 1.13 -> 4.59 W/m^2
- SWCF = -48.6 -> -44.96 W/m^2
- **Link**: #297

## clubb_c8
- **clubb_c8** = 4.2->4.0 
- RESTOM = 1.13 -> 3.23 W/m^2
- SWCF = -48.6 -> -46.5 W/m^2
- **Link**: #298

## Contacts:
@JulioTBacmeister
@PeterHjortLauritzen
@adamrher
@cecilehannay

**Comment by justin-richling** [comment]
ADF for [clubb_gamma_coef vs control](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-gamma.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-gamma.001_1995_2006_vs_f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5.001_1995_2006/)

ADF for [clubb_c8 vs control](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-c8_4.0.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-c8_4.0.001_1995_2006_vs_f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5.001_1995_2006/)
